images/tier1/external.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="18" width="18" viewBox="0 0 18 18">
```

Comment:
I think I'm missing the meaning of this one, maybe the name isn't specific enough. External... what?